src/types/publication.types.ts

```diff
@@ -77,11 +77,25 @@ export interface NpiDiscipline {
   mainDiscipline: string;
 }
 
+export enum DoiRequestStatus {
+  Approved = 'APPROVED',
+  Rejected = 'REJECTED',
+  Requested = 'REQUESTED',
+}
+
+interface DoiRequest {
+  type: string;
+  date: string;
+  status: DoiRequestStatus;
+}
+
 export interface Publication extends BackendType, PublicationFileSet {
   readonly identifier: string;
   readonly createdDate: string;
   readonly owner: string;
   readonly status: PublicationStatus;
+  readonly doiRequest: DoiRequest | null;
+  doiRequested: boolean;
```

Comment:
hva er forskjellen på disse to?

Reply:
fikk du ikke med deg samtalen mellom meg og Jan i dag? 😝

Reply:
Kort forklart: `doiRequested` finnes helt til den er satt til `true`, når den er satt til `true`, så gjør backend et kall mot en annen tjeneste som oppretter en doiRequest. Når doiRequesten er opprettet, finnes ikke feltet `doiRequested` lengre, men vi får et nytt felt (objekt) som heter `doiRequest` 😅

Reply:
trengte en kortversjon :p 
så  verdien for `doiRequested` settes bare på frontend, mens `doiRequest` bare settes på backend?
og alt dette skal sendes til PUT publication, uten noe seperat kall for å requeste doi?

Reply:
Jepp 🙃  Men snakket med Rurik og Terje om dette etter standup i dag, og Rurik mente vi burde ha en separat tjeneste for DOI-requests i fremtiden, men det blir ikke prioritert nå, så foreløpig funker det sånn der 😅